### DISCUSSION TITLE
Screenshot cross-platform with Eto

### DISCUSSION THREAD

**Discussion by bllovesgnr** [picoe/Eto#1922] [Q&A]
I'd like to take screenshot cross-platform with Eto.
After searching the document for a method similar to `CopyFromScreen` on windows, 
I found `Screen.GetImage` that sounds like it can make a screenshot: `Gets a copy of a portion of the screen as an image`
document link: http://pages.picoe.ca/docs/api/html/M_Eto_Forms_Screen_GetImage.htm
so I tried this
`
Image img = screen.GetImage(new RectangleF(new SizeF(Screen.PrimaryScreen.Bounds.Width, Screen.PrimaryScreen.Bounds.Height)));
Console.Write(img);
`

but it doesn't work, the returned image seems to be empty

Any help is appreciated

**Comment by cwensley** [comment]
What platform are you using?  On macOS for example you need to give you app permission to capture the screen other than the current application, which should be a pop up initially but if you said no you need to go to `System Preferences > Security & Privacy > Screen Recording` and check the box next to your application.
